### ISSUE TITLE
Remove accents from search queries

### ISSUE THREAD

**Issue by sampaccoud** [openfun/richie#1347]
## Bug Report

**Problematic behavior/code**
If a course has "électricité" in its title, searching "electricite" does not bring any results whereas "électricité" does.

**Expected Behavior**
Accents in search queries should not be taken into account.

**Steps to Reproduce**

https://user-images.githubusercontent.com/1427165/115736071-49456b80-a38b-11eb-959a-cd9fcee97839.mp4

**Environment**
- Richie version: 2.5.0
- Platform: docker

**Comment by sampaccoud** [comment]
So we just need to apply the same solution to the search query as what was done for autosuggest?

**Comment by mbenadda** [comment]
From a cursory glance, removing the accents is handled by the `asciifolding` filter, which is already present in our analyzers for search. As far as I understand it, accents should already be ignored. I have to work on this to find out.

**Comment by mbenadda** [comment]
I spent more time digging into this. It seems to me like it's not a general problem around accents in words.

There's a specific problem with "électricité" as a word. If you try accented/accentless variations for any other word (any I could find in our course titles anyway), the search results are the same.

I tried looking into the ES side with `explain` and `analyze` queries but it did not yield any results for me. I think we need a pair-programming session to find out what's going on with "électricité" that breaks search.
